### ISSUE TITLE
Separate Library Folders for P1 and P2 Objects

### ISSUE THREAD

**Issue by JonMcPhalen** [parallaxinc/Propeller-Tool#41]
I think it will be helpful to separate the P1 and P2 library folders. When selecting either, the file filter would automatically be set as needed for the folder.

**Comment by PropGit** [comment]
Agreed on separation.  For filter: I think the file filter should be auto-set to the combined P1 & P2 filter when selecting one of these folders, in case someone had misplaced the opposite Propeller type file in there - it'd help them find it / notice it more readily and take steps to solve the mistake.

NOTE: More consideration is in order, but this may require separate library path preferences be made; although with the extension settings as they are, it may not be an issue.  Need to consider the adv/disadv of each before fully implementing.
